src/main/java/frc/robot/commands/ControlPanelCommand.java

```diff
@@ -29,6 +30,6 @@ public void execute() {
                 ColorMatchResult resultColor = controlPanel.getColorMatcher().matchClosestColor(readColor);
                 controlPanel.setDetectedColor(resultColor.color);
         }
-        // controlPanel.getCPMoter().set(VictorSPXControlMode.Velocity, (double)controlPanel.getSpeed());
+        controlPanel.getCPMoter().set(VictorSPXControlMode.Velocity, (double)controlPanel.getSpeed());
```

Comment:
you should call a method on the controlPanel that controls the motor. Don't get the motor and set it here. What if you changed the type of motor or changed how it worked? Then you'll have to change it here as well as any other place that you use it.